src/Rings/mpoly-graded.jl

```diff
@@ -1348,9 +1348,9 @@ end
 
 
 @doc raw"""
-  vector_space(K::Field, polys::Vector{T}; target = nothing) where {T <: MPolyRingElem}
+    vector_space(K::Field, polys::Vector{T}; target = nothing) where {T <: MPolyRingElem}
 
-Return a `K`-vector space `V` and an epimorphism from `V` onto the `K`-vector
+Return a `K`-vector space `V` and an isomorphism from `V` to the `K`-vector
```

Comment:
Caution: You should not change the (user-visible) behaviour of vector_space(Field, Polys) without looking at the other functions vector_space(Field, something), like vector_space(Field, MPolyQuoLocRing). 
This needs to be consistent in behaviour!!! You are only tweaking one function here.

The main application is to pass from a zero-dimensional quotient ring R/I or slice of a graded ring R_d, which of course happens to carry the structure of a finite dimensional vector space, to an object of type vector space by using a monomial basis of the vector space. However, if this family of functions is provided to the user, it must be consistent in all instances.

Reply:
I'm sorry, but I don't understand what should change here.
This is editing the doc string of the function `vector_space(::Field, ::Vector{<:MPolyRingElem})`. The other `vector_space` functions that seem related are:
```
vector_space(K::Field, Q::MPolyQuoRing)
vector_space(kk::Field, W::Oscar.MPolyQuoLocRing{<:Field, <:FieldElem, <:MPolyRing, <:MPolyRingElem, <:Oscar.MPolyComplementOfKPointIdeal}; ordering) 
vector_space(kk::Field, W::Oscar.MPolyQuoLocRing; ordering)
```
Neither of them is documented. Looking at the implementation, I assume that all of them return an isomorphic vector space and not just one that maps surjectively onto the quotient ring. If this assumption is correct, then this pull request will actually increase consistency.

Reply:
Sorry, I abused "changes requested" to buy the time for checking myself that everything is consistent. I should have sent an e-mail instead.
I am now convinced that this does not break anything and does not cause anything inconsistent.